airflow/providers/jira/sensors/jira.py

```diff
@@ -68,7 +68,7 @@ class JiraTicketSensor(JiraSensor):
 
     :param jira_conn_id: reference to a pre-defined Jira Connection
     :type jira_conn_id: str
-    :param ticket_id: id of the ticket to be monitored
+    state:param ticket_id: id of the ticket to be monitored
```

Comment:
```suggestion
    :param ticket_id: id of the ticket to be monitored
```
Travis is sad without this change.

Reply:
Nice catch, thank you! This snuck in somehow. Just pushed up a change.